src/components/emotionStyles.js

```diff
@@ -0,0 +1,76 @@
+import { NEUTRALS, PURPLES } from '../colors/src/Colors';
+
+const emotionStyles = {
```

Comment:
This file is named `emotionStyles`, but it is specific to the `react-select` dependency. I'd prefer using a file name that matches functionality more closely.

Similarly, I'd prefer a different file path. What do you think about `src/style/selectStyles.js` or `src/style/datetimeStyles.js`?

Reply:
I'll go with selectStyles.js